### PR TITLE
[TASK] #102799 - TYPO3_CONF_VARS.GFX.processor_stripColorProfileParameters option added

### PR DESCRIPTION
**Pull request by brotkrueml**
Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/852
Releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/Configuration/Typo3ConfVars/GFX.rst [hunk 1]
```diff
@@ -183,16 +183,53 @@ processor_stripColorProfileByDefault
...
+
+    :type: string
+
+    This option expected a string of command line parameters. The defined
+    parameters had to be shell-escaped beforehand, while the new option
+    :ref:`typo3ConfVars_gfx_processor_stripColorProfileParameters` expects
+    an array of strings that will be shell-escaped by TYPO3 when used.
+
+    The existing configuration will continue to be supported. Still, it is
+    suggested to use the new configuration format, as the Install Tool is
```
Comment (garvinhicking) [inline Documentation/Configuration/Typo3ConfVars/GFX.rst:201]: `:guilabel:` here for "Install Tool"?

File: Documentation/Configuration/Typo3ConfVars/GFX.rst [hunk 2]
```diff
@@ -183,16 +183,53 @@ processor_stripColorProfileByDefault
...
+processor_stripColorProfileParameters
+=====================================
+
+.. confval:: $GLOBALS['TYPO3_CONF_VARS']['GFX']['processor_stripColorProfileParameters']
+
+   :type: array of strings
+   :Default: :php:`['+profile', '*']`
+
+   Specifies the parameters to strip the profile information, which can reduce
    thumbnail size up to 60KB. Command can differ in IM/GM, IM also know the
```
Comment (garvinhicking) [inline Documentation/Configuration/Typo3ConfVars/GFX.rst:228]:
```suggestion
   thumbnail size up to 60KB. Command can differ in IM/GM, IM also knows the
```

File: Documentation/Configuration/Typo3ConfVars/GFX.rst (hunk 1 above)
Comment (garvinhicking) [reply Documentation/Configuration/Typo3ConfVars/GFX.rst:201]: Ah, so it only applies to "real" modules and not the "stand-alone" module that the Install Tool is. Understood, thanks.

**Review by garvinhicking — APPROVED** [review]

File: Documentation/Configuration/Typo3ConfVars/GFX.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/Configuration/Typo3ConfVars/GFX.rst:201]: Yup, the guilabel should show, that there is something on the GUI.

**Pull request merged**